packages/venia-concept/src/components/MiniCart/__stories__/Kebab.js

```diff
@@ -13,15 +13,9 @@ const styles = {
     display: 'grid'
 };
 
-stories.add('Kebab Closed', () => (
+stories.add('Kebab', () => (
     <div style={styles}>
-        <Kebab isOpen={false} />
```

Comment:
Kebab doesn't need to accept an `isOpen` prop. Its state is internal.

Reply:
I'll take note, but it`s outdated conversation.

Reply:
Oh, this was just me leaving a comment explaining why I removed the line :)

Reply:
Oh, sorry, but I will take note anyway :)